src/@lekoarts/gatsby-theme-minimal-blog/components/seo.tsx

```diff
@@ -19,27 +21,29 @@ const Seo = ({
   children = null,
   canonicalUrl = ``,
 }: SEOProps) => {
-  const site = useSiteMetadata()
-
+  const { language } = useI18next();
+  const site = useSiteMetadata();
+  const { t } = useTranslation();
   const {
-    siteTitle,
-    siteTitleAlt: defaultTitle,
+    // siteTitle, we can pass the i18n id here as well.
+    // siteTitleAlt: defaultTitle,
     siteUrl,
     siteDescription: defaultDescription,
     siteImage: defaultImage,
     author,
     siteLanguage,
-  } = site
+  } = site;
 
   const seo = {
-    title: title ? `${title} | ${siteTitle}` : defaultTitle,
```

Comment:
Also could we please mark seo related variables with `seo_` prefix?

Reply:
@romanr111  for  the translation keys right? 

```
site_title => **seo_site_title**

```

Reply:
yea, right!